crates/matrix-sdk-base/src/client.rs

```diff
@@ -496,13 +498,18 @@ impl BaseClient {
                                 SyncMessageLikeEvent::Original(original_event),
                             ) => match &original_event.content.msgtype {
                                 MessageType::VerificationRequest(_) => {
-                                    Box::pin(self.handle_verification_event(e, room.room_id()))
-                                        .await?;
+                                    if !ignore_verification_requests {
+                                        Box::pin(self.handle_verification_event(e, room.room_id()))
+                                            .await?;
+                                    }
```

Comment:
Shouldn't we ignore all verification events? I guess the crypto crate already does so if it doesn't receive an `m.key.verification.request` event, but I think it'll log a bunch of warnings in that case.

Reply:
Isn't that what I'm doing below? I'm not sure I fully understand the underlying logic, but I think this branch takes the verification requests and the one below any other verification event.

Reply:
Ah, how did I miss that one. You are indeed right.

Can we then rename the flag and setting? Since we indeed don't just ignore the requests `ignore_verification_events` might make more sense.